src/components/nutrition/label/NutritionLabel.tsx

```diff
@@ -0,0 +1,172 @@
+    margin: "1pt"
+}));
+
+const CalorieRow = styled(Box)(() => ({
+    padding: 0,
+    margin: "3pt 0",
+    lineHeight: "8pt",
+    height: "8pt"
+}));
+
+const CaloriesLabel = styled(Typography)(() => ({
+    fontSize: "8pt",
+    fontWeight: 700,
+    margin: "0 0 0 1pt",
+    float: "left"
+}));
+
+const CaloriesValue = styled(Typography)(() => ({
+    fontSize: "8pt",
+    fontWeight: 100,
+    margin: "0 0 0 2pt",
+    float: "left"
+}));
+
+const CaloriesFromFat = styled(Typography)(() => ({
+    fontSize: "8pt",
+    fontWeight: 100,
+    display: "inline-flex",
+    margin: "0 1pt 0 auto",
+    float: "right"
+}));
+
+const PercentDailyValueLabel = styled(Typography)(() => ({
+    lineHeight: "7pt",
+    fontSize: "8pt",
+    fontWeight: 100,
+    textAlign: "right",
+    margin: "2pt 0 5pt",
+}));
+
+const DailyValues = styled(Typography)(() => ({
+    lineHeight: "7pt",
+    fontSize: "7pt",
+    fontWeight: 100,
+    textAlign: "left",
+    margin: "2pt 0 5pt"
+}));
+
+function NutritionLabel({
+    backgroundColor, servingText, calories, caloriesFromFat, totalFat,
+    saturatedFat, transFat, cholesterol, sodium, totalCarbohydrate,
+    potassium, dietaryFiber, sugars, protein, vitamins, width
+}: {
+    backgroundColor?: string,
+    servingText: string,
+    calories: number,
+    caloriesFromFat: number,
+    totalFat: number,
+    saturatedFat: number,
+    transFat: number,
+    cholesterol: number,
+    sodium: number,
+    potassium: number,
+    totalCarbohydrate: number,
+    dietaryFiber: number,
+    sugars: number,
+    protein: number,
+    vitamins: Array<string>,
+    width?: string,
+    //?=maybe for type
+}) {
+    return (
+        <LabelContainer backgroundColor={backgroundColor} width={width}>
+            <Title>Nutrition Facts</Title>
+            <ServingInfo>
+                {`Serving Size ${servingText}`}
+            </ServingInfo>
+
+            <SeparatorBar height={"7pt"} color={backgroundColor}/>
+
+            <AmountPerServing>
+                {`Amount Per Serving`}
+            </AmountPerServing>
+
+            <SeparatorBar height={"0.5pt"} color={backgroundColor}/>
+
+            <CalorieRow>
+                <CaloriesLabel>
+                    {`Calories `}
+                </CaloriesLabel>
+                <CaloriesValue>
+                    {calories||0}
+                </CaloriesValue>
+                {caloriesFromFat > 0 && (
+                    <CaloriesFromFat>
+                        {`Calories from Fat ${caloriesFromFat}`}
+                    </CaloriesFromFat>
+                )}
+            </CalorieRow>
+
+            <SeparatorBar height={"3pt"} color={backgroundColor}/>
+
+            <PercentDailyValueLabel>
+                {`% Daily Value*`}
+            </PercentDailyValueLabel>
+
+            <SeparatorBar height={"0.5pt"} color={backgroundColor}/>
+
+            <NutrientRow label={"Total Fat"} value={totalFat||0} unit="g" boldLabel={true} recommended={78} />
```

Comment:
FYI, in React `boldLabel={true}` can also be written as just `boldLabel` for brevity, and omitted for the false case. Spelling it out is fine too, though.